### DISCUSSION TITLE
[Backend] Quarkus + Kotlin

### DISCUSSION THREAD

**Discussion by u-ways** [realworld-apps/realworld#697] [Added implementations]
I am currently working on [kotlin-quarkus-realworld-example-app](https://github.com/U-ways/kotlin-quarkus-realworld-example-app). Only the Comment section left, then I will run the postman tests, complete the README file, and do some minor clean up. I followed a TDD approach as best as my time allows. Feel free to provide any feedback or suggest any improvements!

**Comment by u-ways** [comment]
This is now complete, just some minor clean ups and a comprehensive README that fits the community guidelines are left to do 🎉

**Reply by geromegrignon** [reply]
Awesome!
